README.md

```diff
@@ -147,7 +147,7 @@ bucket.
 
 | Version | Cloud TPU VMs Wheel |
```

Comment:
since this part is the only one part include wheels for TPU/GPU, let's modify name from `Cloud TPU VMs Wheel` to `Cloud TPU/GPU VMs Wheel`